ideacrew/gluedb
# fixing edi transaction reinstate filter to adjust pagination with click

https://www.pivotaltracker.com/story/show/184539486

Previously:
The pagination did not update when the checkbox was clicked

Now:
When checkbox is clicked, the pagination updates to reflect new counts

QA Notes: 
This change adjusts the search partial so need to ensure all other pages that use this partial are unchanged as well.
